src/tv2-common/inewsConversion/converters/ParseCue.ts

```diff
@@ -918,6 +926,17 @@ function parseDesignBg(cue: string[], config: TV2BlueprintConfig): CueDefinition
 	})
 }
 
+function parseTelemetricsCue(cue: string[]): CueDefinitionTelemetrics {
+	const presetIdentifier: number = Number(cue[0].match(/\d+/))
+	const time: Pick<CueDefinitionBase, 'start' | 'end'> = cue[1] ? parseTime(cue[1]) : { start: { seconds: 0 } }
+	return {
+		type: CueType.Telemetrics,
+		iNewsCommand: 'Telemetrics',
```

Comment:
After I was 3/4 through the implementation it hit me that I had used the name `Telemetrics` throughout the code and I wasn't sure if it should actually have been `Robot` instead.

For example, this method I think would make more sense to be called `parseRobotCue` since the cue written in iNews is called `ROBOT`. And then the type should be `CueType.Robot` instead of `CueType.Telemetrics`.

Big question is: Where do we go from Robot to Telemetrics? Do we make Robot blueprint pieces and timeline objects that just now that they are supposed to be send to a Telemetrics device or should we make the 'cut' somewhere else? (It would mean renaming `EvaluateCueTelemtrics` to `EvaluateCueRobot` and similar things)